### PR TITLE
Add okadash for Mac 1.8.0

### PR DESCRIPTION
**Pull request by okadato623**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/okadash.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'okadash' do
+  version '1.8.0'
+  sha256 '063c70ba0419a3d0f6509ba408fa041589f40a9a10377ea18fa526046df5e6f6'
+
+  url 'https://github.com/konoyono/okadash/releases/download/1.8.0/okadash-1.8.0-mac.zip'
+  appcast 'https://github.com/konoyono/okadash/releases/latest'
```
Comment (miccal) [inline Casks/okadash.rb:6]: Please change these to:
```
url "https://github.com/konoyono/okadash/releases/download/#{version}/okadash-#{version}-mac.zip"
appcast 'https://github.com/konoyono/okadash/releases.atom'
```

Comment (konoyono) [reply Casks/okadash.rb]: Thanks for commenting. Fixed them.

**Comment by ran-dall** [comment]
@vitorgalvao How do we feel about casks that aren't meeting the new notability requirements, but meet the old requirements? Are still merging them?

**Comment by vitorgalvao** [comment]
@ran-dall We’re not. I’ve already refused a few under the new requirements.

**Comment by vitorgalvao** [comment]
Thank you for the submission, @konoyono, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> + The Cask is too obscure. Examples:
>   + A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 30 forks, 30 watchers, and 75 stars).
>   + [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021).

**Pull request closed without merging**